exampleCourse/questions/template/checkbox/feedback/question.html

```diff
@@ -0,0 +1,17 @@
+<pl-question-panel>
+  <p>Which of the following cities are located in {{params.country}}?</p>
+</pl-question-panel>
+
+<pl-checkbox
+  answers-name="cities"
+  number-answers="6"
+  partial-credit="true"
+  min-correct="2"
+>
+  {{#params.correct_cities}}
+    <pl-answer correct="true">{{.}}</pl-answer>
+  {{/params.correct_cities}}
+  {{#params.distractor_cities}}
+    <pl-answer correct="false" feedback="{{city}} is in {{country}}.">{{city}}</pl-answer>
+  {{/params.distractor_cities}}
```

Comment:
Note that I am still using Markdown loops here; I think this is generally cleaner than trying to assign to 10 different things in `data["params"]`, specially since all the Python code is already dealing with lists.

Reply:
Ok, I guess I reluctantly agree with you 😅. Is there any way we can add a comment to indicate what the mustache loop is doing?

Reply:
```
{{! Comment }}
```
https://mustache.github.io/mustache.5.html

Reply:
Comment added in a947452.

Reply:
Excellent! I proposed some more verbose comments to try and explain to a reader what's going on. Feel free to edit or discard as you like.

Reply:
Added in 2a4ff56!